scenes/enemies/Sheep/WasAttacked.gd

```diff
@@ -0,0 +1,18 @@
+extends ConditionLeaf
+
+var was_attacked = false
+var attacker_name = null
+func tick(actor: Node, blackboard: Blackboard):
+	if not actor.synchronizer.got_hurt.is_connected(_on_got_hurt):
```

Comment:
It should be getting called every physics tick, yes.

I mostly copied the code style from the examples they provided. I think it needs to get called every tick since at certain points, you will disconnect the signal. These behavior nodes within Beehave are long lived. They aren't recreated every time so I think since there is disconnecting and reconnecting happening based on tree changes, I need it to stay in the `tick(...)` function.

Reply:
ok closing, it is indeed in line with style of behave